src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -70,7 +71,13 @@ public void shooterInit() {
 
     pivotMotor.getConfigurator().apply(ArmTunerConstants.pivotMotionMagicConfigs);
     pivotMotor.getConfigurator().apply(ArmTunerConstants.pivotPIDConfigs);
-    // TODO apply SoftwareLimitSwitchConfigs to pivotMotor to prevent the robot arm from crashing
+
+    SoftwareLimitSwitchConfigs pivotLimits = new SoftwareLimitSwitchConfigs();
+    pivotLimits.ForwardSoftLimitThreshold = 50.0 / 360.0; // maximum of 90 degrees
+    pivotLimits.ReverseSoftLimitThreshold = 0;
```

Comment:
I don't think this is right. First things first, the limit is actually a minimum of like 30 ish and I set the setpoint to just below 30 so it doesn't crash into anything when it's dropping down. And second of all, I wouldn't think that you would need to offset by the minimum to get the rotations value since the wrist calibration is accommodates the zero point correctly and knows setting it at an angle should be ~30 degrees.